proptest/src/test_runner/rng.rs

```diff
@@ -330,6 +406,14 @@ impl TestRng {
         TestRng::from_seed_internal(Seed::from_bytes(algorithm, seed))
     }
 
+    /// dumps the bytes obtained from the RNG so far (only works if the RNG is set to Recorder)
+    pub fn bytes_used(&self) -> Vec<u8> {
```

Comment:
Wouldn't it be better to return `Option<Vec<u8>>` here or something?

Reply:
I couldn't feel of a use-case where you wouldn't `unwrap()` if you did pass an `Option`. Because in the end you know that you've generated a value before calling this one. Wdyt?

Reply:
Yeah, I guess that's reasonable